SubmitHGCalPGun.py

```diff
@@ -362,6 +364,9 @@ def submitHGCalProduction():
                 if opt.gunMode != 'physproc':
                     s_template=s_template.replace('MAXTHRESHSTRING',"Max"+str(opt.gunType))
                     s_template=s_template.replace('MINTHRESHSTRING',"Min"+str(opt.gunType))
+                else:
```

Comment:
I don't think we need these additional placeholders `MINXTHRESHSTRING` and `MAXTHRESHSTRING`. We can just reuse `DUMMYTHRESHMIN/MAX` that are replaced in https://github.com/CMS-HGCAL/reco-prodtools/blob/09bb40249fee2d8d45b4c7e753a6a9c4c8f8aacd/SubmitHGCalPGun.py#L359

Reply:
This should read that we don't need the whole `else` clause here :-)

Reply:
Removed the `else:` block in the last commit and reused the threshold parameters.